### PR TITLE
modify non_ubuntu_os_pkg and  nonubuntu_os_other_pkg

### PR DESCRIPTION
**Pull request by junxiawang**
@tingtli 
link for doc:
http://10.4.30.1/install/html/pkg/guides/admin-guides/manage_clusters/common/deployment/additionalpkg/

### CODE REVIEW

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/additionalpkg/nonubuntu_os_pkg.rst [hunk 1]
```diff
@@ -80,7 +80,7 @@ Note: the trailing **"#"** character at the end of the line. It is important to
 Group/Pattern Names
 '''''''''''''''''''
 
-It is only supported for stateful deployment.
+It is only supported for statefull deployment.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/additionalpkg/nonubuntu_os_pkg.rst]: I believe "stateful" is the correct term that xCAT invented... 
I don't know the history, but diskful and stateful do not have two "l"s

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/additionalpkg/nonubuntu_os_pkg.rst [hunk 2]
```diff
@@ -99,7 +99,7 @@ You can specify in this file the group/pattern names by adding a **'@'** and a s
 Remove RPMs After Installing
 ''''''''''''''''''''''''''''
 
-It is only supported for stateful deployment.
+It is only supported for statefull deployment.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/additionalpkg/nonubuntu_os_pkg.rst]: Same comment as above.

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/additionalpkg/nonubuntu_os_pkg.rst (hunk 1 above)
Comment (daniceexi) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/additionalpkg/nonubuntu_os_pkg.rst]: I guess the history of using 'stateful/stateless' was trying to make the term 'statelite'. If we want to remove 'statelite' from xcat, I prefer to change back to use the more understanable term 'diskful/diskless'.

Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/additionalpkg/nonubuntu_os_pkg.rst]: Yea,  I see a whole bunch of stateful and statelite mentioned interchangeably.. I would be up for going with only diskful or disks less in our terminology

**Comment by daniceexi** [comment]
This pull request has been reviewed by @whowutwut throughly, it OK for merge.

**Pull request merged**